File: sdk/WinPhone_sdk/UMSAgent/MyObject/OnLineConfig.cs
/**
 * Cobub Razor
 *
 * An open source analytics windows phone sdk for mobile applications
 *
 * @package		Cobub Razor
 * @author		WBTECH Dev Team
 * @copyright	Copyright (c) 2011 - 2012, NanJing Western Bridge Co.,Ltd.
 * @license		http://www.cobub.com/products/cobub-razor/license
 * @link		http://www.cobub.com/products/cobub-razor/
 * @since		Version 0.1
 * @filesource
 */

using System;
using System.Net;
using System.Windows;
using System.Windows.Controls;
using System.Windows.Documents;
using System.Windows.Ink;
using System.Windows.Input;
using System.Windows.Media;
using System.Windows.Media.Animation;
using System.Windows.Shapes;

namespace UMSAgent.MyObject
{
    public class OnLineConfig
    {
        private string app_key;
        public string appkey
        {
            set { this.app_key = value;}
            get {return this.app_key;}
        }
    }
}

File: web/application/models/redis_service/onlineconfig.php
<?php
/**
 * Cobub Razor
 *
 * An open source mobile analytics system
 *
 * PHP versions 5
 *
 * @category  MobileAnalytics
 * @package   CobubRazor
 * @author    Cobub Team <open.cobub@gmail.com>
 * @copyright 2011-2016 NanJing Western Bridge Co.,Ltd.
 * @license   http://www.cobub.com/docs/en:razor:license GPL Version 3
 * @link      http://www.cobub.com
 * @since     Version 0.1
 */

/**
 * OnlineConfig Model
 *
 * @category PHP
 * @package  Model
 * @author   Cobub Team <open.cobub@gmail.com>
 * @license  http://www.cobub.com/docs/en:razor:license GPL Version 3
 * @link     http://www.cobub.com
 */
class OnlineConfig extends CI_Model
{



    /** 
     * Online config 
     * OnlineConfig load 
     * 
     * @return void 
     */
    function OnlineConfig()
    {
        parent::__construct();
        $this -> load -> database();
        $this -> load -> library('redis');
    }
    
    /** 
     * Get productid 
     * GetProductid function 
     * 
     * @param string $key key 
     * 
     * @return string 
     */
    function getProductid($key)
    {
        $query = $this -> db -> query("select product_id from " . $this -> db -> dbprefix('channel_product') . " where productkey = '$key'");
        if ($query != null && $query -> num_rows() > 0) {
            $productid = $query -> first_row() -> product_id;
            return $productid;
        }
        return null;
    }
    
    /** 
     * Get config message 
     * GetConfigMessage function 
     * 
     * @param string $productid productid 
     * 
     * @return array 
     */
    function getConfigMessage($productid)
    {
        $query = $this -> db -> query("select * from " . $this -> db -> dbprefix('config') . " where product_id = '$productid'");
        if ($query != null && $query -> num_rows() > 0) {
            return $query -> first_row();

        }
        return null;
    }

}
